src/main/java/net/dv8tion/jda/core/managers/GuildController.java

```diff
@@ -603,6 +605,8 @@ protected void handleResponse(Response response, Request<Void> request)
 
         Checks.notNegative(delDays, "Deletion Days");
 
+        Checks.notBiggerThan(delDays, 7, "Deletion Days");
```

Comment:
Not entirely sure if this would be the best approach, maybe change it to a simple if statement to avoid creating a new method on `Checks.java`.

Reply:
yea, we have `Checks.check(Predicate check, String message)` iirc

Reply:
An alternative is `Checks.notPositive(delDays - 7, ...)`